src/cache/cache.service.ts

```
@@ -0,0 +1,251 @@
+      const v = await this.withTimeout(this.store.incr(this.versionKey(namespace)));
+      this.onOpSuccess();
+      this.logger.debug(`cache INCR ns=${namespace} v=${v} caller=${caller}`);
+    } catch (err) {
+      this.onOpFailure();
+      this.logger.error(
+        `cache INCR failed ns=${namespace} caller=${caller}: ${err.message}`,
+      );
+    }
+  }
+
+  private writeCacheEntry(entryKey: string, result: unknown, ttlSeconds: number) {
```

ESLint

[error] Replace `entryKey:·string,·result:·unknown,·ttlSeconds:·number` with `⏎····entryKey:·string,⏎····result:·unknown,⏎····ttlSeconds:·number,⏎··`

(prettier/prettier)